data/modules/TradeShips/Debug.lua

```diff
@@ -105,8 +105,10 @@ debugView.registerTab('debug-trade-ships', function()
 			property("Total flow", string.format("%.2f ship/hour", Core.params.total_flow))
 			ui.sameLine()
 			property("Last spawn interval", ui.Format.Duration(Core.last_spawn_interval))
-			ui.sameLine()
-			property("Lawlessness", string.format("%.4f", Game.system.lawlessness))
+			if Game.system then
```

Comment:
One more point - maybe even disable the entire tab during hyperspace? iI seems that nothing but the log is relevant there anymore (and the log is not particularly relevant).

Reply:
I think there is no harm if the tab stays during hyperspace. It's short sequence anyway. Maybe log should be cleared when player is in hyperspace, because ships that are there are deleted by that point?

Reply:
How to say, for example, a flight on Coronatrix 36 light years lasts 45 seconds of real time, enough to go to 'Local routes', click somewhere and crash the game. Do we have a good reason to show this tab in hyperspace?

Reply:
`TradeShips` debug tab now will be disabled in the hyperspace.

Reply:
Thanks!